content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -27,6 +27,7 @@ weight: 10
 
 ## Installation
 
+- **Added** Added the [operator controller](/docs/ops/setup/standalone-operator/) for dynamic updates to an Istio installation.
 - **Removed** the `proxy_init` Docker image. Instead, the `istio-init` container reuses the `proxyv2` image.
 - **Updated** the base image to `ubunutu:bionic`.
```

Comment:
Do we want to mention that istioctl is now default recommended method for new installs?